DFC.ServiceTaxonomy.GraphSync/GraphSyncers/Interfaces/IContentPartGraphSyncer.cs

```diff
@@ -14,5 +16,12 @@ public interface IContentPartGraphSyncer
             IMergeNodeCommand mergeNodeCommand,
             IReplaceRelationshipsCommand replaceRelationshipsCommand,
             ContentTypePartDefinition contentTypePartDefinition);
+
+        Task<bool> VerifySyncComponent(
+            ContentItem contentItem,
```

Comment:
What do you think of reordering the parameters to `content, conterntTypePartDefinition, node, relationships, destNodes`, so that the content and graph elements are grouped together?